bench/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+
+include(../cmake/GoogleBenchmark.cmake)
+
+
+add_custom_target(Bench_MooringConfigFiles)
+
+add_custom_command(TARGET Bench_MooringConfigFiles PRE_BUILD
+                   COMMAND ${CMAKE_COMMAND} -E
+                        copy_directory ${CMAKE_SOURCE_DIR}/bench/Mooring ${CMAKE_CURRENT_BINARY_DIR}/Mooring)
+if(WIN32)
+    add_custom_command(TARGET Bench_MooringConfigFiles POST_BUILD
+                       COMMAND ${CMAKE_COMMAND} -E copy $<TARGET_FILE:moordyn> ${CMAKE_CURRENT_BINARY_DIR}/)
+endif()
+
+function (add_bench bench_name src)
+    add_executable(${bench_name} ${src})
+    add_dependencies(${bench_name} Bench_MooringConfigFiles)
+    target_link_libraries(${bench_name} benchmark::benchmark)
+    target_link_libraries(${bench_name} moordyn)
+    target_include_directories(${bench_name} PUBLIC ${CMAKE_BINARY_DIR}/)
+endfunction()
+
+add_bench(WaveBenchmark WaveBench.cpp)
+add_bench(MoordynBenchmark MDBench.cpp)
```

Comment:
How do I run the benchmarks? Shall I run the executables manually? If that is the case, maybe we can [create a custom target named run_benchmarks](https://cmake.org/cmake/help/latest/command/add_custom_target.html), depending on both WaveBenchmark and MoordynBenchmark

What do you think?

Reply:
Easy enough, done.